src/content/cloud/okteto-cli.mdx

```diff
@@ -71,10 +71,10 @@ The installation job also populates the following environment variables. You can
 - `BUILDKIT_HOST`: the address of the [Okteto Cloud buildkit instance](/docs/cloud/build/). You can use it to build your images with any CLI tool that supports Buildkit builds.
 
 In order to refer to images in the `build` section of the Okteto Manifest, you can use the following environment variables:
-- `${OKTETO_BUILD_<service>_TAG}`: SHA for the last image build at the registry
+- `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference (using its SHA to force redeploy)
 - `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed
 - `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)
```

Comment:
Is the underscore part of the value? (_service)

Reply:
fixed